examples/simple/tsconfig.json

```diff
@@ -1,5 +1,10 @@
 {
     "compilerOptions": {
-        "declaration": true
+        "declaration": true,
+        "baseUrl": ".",
+        "paths": {
+            "@myorg/*": ["../*"]
```

Comment:
It feels like this shouldn't be needed since tsc will find it inside node_modules.

Reply:
Oh. its probably for the IDE to locate these properly/

Reply:
it's needed for the editor to avoid red squiggles. I'll add a comment